frontend/mulighetsrommet-cms/schemas/tiltakstype.js

```diff
@@ -44,15 +52,43 @@ export default {
       title: "Varighet",
       type: "string",
     },
+    //TODO skal kunne legge til flere lenker og filer
     {
       name: "regelverkFil",
       title: "Regelverk fil",
       type: "file",
     },
+    {
+      name: "regelverkFilNavn",
+      title: "Navn til fil",
+      description: "Hvilket navn skal vises til filen?",
+      type: "string",
+      validation: (Rule) =>
+        Rule.custom((field, context) =>
+          context.document.regelverkFil && field === undefined
+            ? "This field must not be empty."
```

Comment:
All tekst i løsningen er på norsk. Er det en grunn til at du plutselig går for en engelsk feilmelding?